src/reports/models/reportsManager.js

```diff
@@ -93,8 +93,8 @@ module.exports.postStats = async (report, stats) => {
     }
     await databaseConnector.updateReport(report.test_id, report.report_id, { phase: report.phase, last_updated_at: statsTime });
     report = await module.exports.getReport(report.test_id, report.report_id);
```

Comment:
maybe here change the order of line 95/96
then the report will have the benchmark data

Reply:
but I need the updated report